en_US/data-integration/data-bridge-kafka.md

```diff
@@ -157,9 +167,9 @@ You can also click **Integration** -> **Flow Designer** to view the topology. Th
 
 ![Kafka_producer_bridge](./assets/Kafka_producer_bridge.png)
 
-### Configure Kafka Dynamic Topics
+### Use Environment Variables in Rule SQL
```

Comment:
This heading is better to be kept as the old one.
Use env for Kafka topics is one special case which is more suitable as a tip.
A more common use case is to map mqtt topic to Kafka topic, or map a payload field to Kafka topic